### PR TITLE
WEBEU2 - User Interface - Sulaiman Abidemi 🇳🇬

### PR DESCRIPTION
**Pull request by abidex4yemi**
### What does this PR Do
* Create all HTML markup for the great idea project 

* List of the task to be completed

- add header markup
 - add nav markup
 - add hero section markup
 - add company-info markup
 - add footer markup

Screenshot 2019-04-23 at 3 59 05 PM

### CODE REVIEW

File: great-idea-website/index.html [hunk 1]
```diff
@@ -1,63 +1,144 @@
...
-
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-
   <title>Great Idea!</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Lato|Playball|Roboto" rel="stylesheet">
```
Comment (maximesalomon) [inline great-idea-website/index.html:7]: Good job using a google font

File: great-idea-website/index.html [hunk 2]
```diff
@@ -1,63 +1,144 @@
...
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
+<body>
+  <header class="header">
```
Comment (maximesalomon) [inline great-idea-website/index.html:16]: good use of semantic header here

File: great-idea-website/index.html [hunk 3]
```diff
@@ -1,63 +1,144 @@
...
+    <div class="container">
+      <div class="header-content">
+        <nav class="main-nav">
+          <ul class="nav-list-container">
+            <li class="nav-list-item">
+              <a href="index.html" class="nav-list-item-link">Services</a>
+            </li>
+            <li class="nav-list-item"><a href="#!" class="nav-list-item-link">Product</a></li>
```
Comment (maximesalomon) [inline great-idea-website/index.html:24]: You don't need nav-list-item-link here you could use the parent class with the a tag

File: great-idea-website/index.html [hunk 4]
```diff
@@ -1,63 +1,144 @@
...
+          </ul>
+        </nav>
+
+        <a href="index.html" class="logo-link">
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </a>
+      </div>
+    </div>
+  </header>
+  <!--/.header-->
```
Comment (maximesalomon) [inline great-idea-website/index.html:38]: Good use of comment!

File: great-idea-website/index.html [hunk 5]
```diff
@@ -1,63 +1,144 @@
...
+
```
Comment (maximesalomon) [inline great-idea-website/index.html:39]: You might want to add a `<main>` tag here to add some semantic

File: great-idea-website/index.html [hunk 6]
```diff
@@ -1,63 +1,144 @@
...
+  <section class="hero">
+    <div class="container">
+      <div class="hero-content">
+        <p>
```
Comment (maximesalomon) [inline great-idea-website/index.html:43]: You should be using a h1 here

File: great-idea-website/index.html [hunk 7]
```diff
@@ -1,63 +1,144 @@
...
+            <p>
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+              Suspendisse
+              eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+          <!--/.company-info-services-->
+
+          <div class="company-info-product info-section">
+            <h1 class="company-info-heading">Product</h1>
```
Comment (maximesalomon) [inline great-idea-website/index.html:93]: You can use only one h1 per page. You should be using a h3 here.

File: great-idea-website/index.html [hunk 8]
```diff
@@ -1,63 +1,144 @@
...
+          <div class="company-info-vision info-section">
+            <h1 class="company-info-heading">Vision</h1>
+            <p>
+              Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+              Suspendisse
+              eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+            </p>
+          </div>
+          <!--/.company-info-product-->
+        </div>
```
Comment (maximesalomon) [inline great-idea-website/index.html:111]: You should close the section here and open a footer for below

File: great-idea-website/css/index.css [hunk 9]
```diff
@@ -3,64 +3,376 @@
...
+	margin-left: 5rem;
+	text-align: center;
+}
+
+.hero-chant {
+	font-family: 'Playball', cursive;
+	color: #010A26;
+	display: block;
+	font-size: 5rem;
+	text-transform: uppercase;
```
Comment (maximesalomon) [inline great-idea-website/css/index.css:236]: good use of CSS3 text-transform

**Review by maximesalomon — APPROVED** [review]
Good job Sulaiman! You also completed some stretch keep going :)

File: great-idea-website/index.html (hunk 3 above)
Comment (abidex4yemi) [reply great-idea-website/index.html:24]: yes that's true but i'm using BEM css so nesting will cause style dependencies.

File: great-idea-website/index.html (hunk 8 above)
Comment (abidex4yemi) [reply great-idea-website/index.html:111]: oh good, yes correct!. thanks

**Pull request merged**